lib/rubygems/security/signer.rb

```diff
@@ -29,6 +29,15 @@ class Gem::Security::Signer
 
   attr_reader :digest_name # :nodoc:
 
+  ##
+  # Gem::Security::Signer options
+
+  attr_reader :options
+
+  DEFAULT_OPTIONS = {
+    expiration_length_days: 365
```

Comment:
We're defining the default expiration time in a few places. Maybe this constant should come from `ConfigFile::DEFAULT_CERT_EXPIRATION_LENGTH_DAYS ` ?

Reply:
I don't want to make `Gem::Security::Signer` dependant of `Gem::ConfigFile` by requiring it just for a constant. what you think?